dts/bindings/dac/ti,dac-x0501.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2023 Google, LLC.
+# SPDX-License-Identifier: Apache-2.0
+
+include: base.yaml
+
+description: TI DACx0501 12 to 16 bit DAC
+
+compatible: "ti,dac-x0501"
+
+properties:
+  ref-source:
+    type: string
+    required: true
+    enum:
+      - "internal"
+      - "external"
+    description: |
+      DAC voltage reference select: either internal or external
+
+  voltage_mult:
+    type: string
+    enum:
+      - "mul2"
+      - "nop"
+      - "div2"
+    default: "nop"
```

Comment:
any DT binding default must have a rationale in the description

https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values

Reply:
Added

Reply:
it's still not fixed, please look at the link for explanation

Reply:
In that description in the commit, you don't write the justification which you just wrote in this comment, that's what is required, the reasoning for the default value

Reply:
Added now in the source.